modules/osdk-scorecard-store-output.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+:osdk_ver: v1.8.0
+
+[id="osdk-scorecard-store-output_{context}"]
+= Scorecard output storage
```

Comment:
Do we need to add more context about this feature?  Feels like there should be more. I'll have @theishshah look this over as well.

Reply:
@jmrodri @theishshah I am happy to add additional context. I was working primarily off of the [upstream docs ](https://sdk.operatorframework.io/docs/testing-operators/scorecard/custom-tests/#storing-scorecard-test-output), so please let me know what I can add.

Reply:
This feature can also be enabled in the config.yaml via the following:
```
tests:
  - entrypoint:
    - scorecard-test
    - <test-entrypoint>
    image: quay.io/operator-framework/scorecard-test:v1.10.0
    labels:
      suite: <test-suite>
      test: <test-name>
    storage:      
      spec:        
        mountPath:
          path: /test-output
```

Additionally if we can add a line at the beginning stating that users can enable local storage of scorecard results in either the CLI or in the config files.

Thanks.

Reply:
@theishshah I have made the requested changes along with a little bit of clean up. Would you PTAL?